crates/sui-types/src/type_resolver.rs

```diff
@@ -3,17 +3,18 @@
 
 use crate::{
     error::{ExecutionError, SuiError},
-    object::{MoveObject, ObjectFormatOptions},
+    object::MoveObject,
 };
-use move_core_types::{language_storage::TypeTag, value::MoveStructLayout};
+use move_core_types::{annotated_value as A, language_storage::TypeTag, runtime_value as R};
 use move_vm_types::loaded_data::runtime_types::Type;
 
 pub trait LayoutResolver {
-    fn get_layout(
+    fn get_runtime_layout(&mut self, object: &MoveObject) -> Result<R::MoveStructLayout, SuiError>;
```

Comment:
Maybe it will be clear to me later, but do we ever need/use this?

Reply:
We don't as far as I can tell. Will try and remove it!